hooks/post-checkout

```diff
@@ -2,6 +2,8 @@
 
 # stolen from https://gist.github.com/sindresorhus/7996717
 
+echo "Running post-checkout hook..."
```

Comment:
I'm not keen on this verbosity, do you think it adds useful information? I change between branches reasonably frequently (and I sync them up so I'm not triggering yarn install).

Reply:
shared-git-hooks prints something like this whenever it triggers, so I added it for the time being to replace that functionality. I agree that it's probably not necessary for post-checkout and post-merge, but I'd like to keep it in pre-commit because otherwise that would run for pretty long without any indication of what's happening.